src/UglyToad.PdfPig/Geometry/PdfPath.cs

```diff
@@ -5,27 +5,29 @@ namespace UglyToad.PdfPig.Geometry
     using System.Collections.Generic;
     using System.Linq;
     using System.Text;
+    using UglyToad.PdfPig.Core;
 
     /// <summary>
     /// A path in a PDF document, used by glyphs and page content.
     /// </summary>
     public class PdfPath
     {
-        private readonly List<IPathCommand> commands = new List<IPathCommand>();
+        public readonly List<IPathCommand> Commands = new List<IPathCommand>();
         private PdfPoint? currentPosition;
+        internal TransformationMatrix CurrentTransformationMatrix;
 
         internal void MoveTo(decimal x, decimal y)
         {
-            currentPosition = new PdfPoint(x, y);
-            commands.Add(new Move(currentPosition.Value));
+            currentPosition = CurrentTransformationMatrix.Transform(new PdfPoint(x, y));
```

Comment:
So it looks like the test failures are due to `CurrentTransformationMatrix` not being set prior to these calls when it's used by `CharString` operations in font handling text (Type 1 fonts are formed of path operations). I think it would be better to move these transformations out into the `OperationContext` so that `x` and `y` have already been transformed prior to passing into the `MoveTo`, `LineTo`, etc, methods.
To make this easier you could add a `TransformationMatrix CurrentTransformationMatrix { get; }` property to the `IOperationContext` interface.